src/components/ResourceList/ResourceList.tsx

```diff
@@ -8,7 +8,7 @@ import EventListener from '../EventListener';
 import Sticky from '../Sticky';
 import Spinner from '../Spinner';
 import {withAppProvider, WithAppProviderProps} from '../AppProvider';
-import Select, {Option} from '../Select';
+import Select, {SelectOption} from '../Select';
```

Comment:
I tried changing just the import statement to `import Select, {Option as SelectOption} from '../Select';` but that did not work.

`yarn run typedoc` compiled correctly but when viewing the types it just said `SelectOption[]` instead of exposing what that intersection type referred to  (i.e. `string | StrictOption[]`) like on production.